platforms/js/build_js.py

```diff
@@ -140,6 +140,7 @@ def get_cmake_cmd(self):
                "-DBUILD_PACKAGE=OFF",
                "-DBUILD_TESTS=OFF",
                "-DBUILD_PERF_TESTS=OFF"]
+
```

Comment:
Please remove extra changes from this file

Reply:
Done